calico-enterprise/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -160,6 +163,13 @@ To re-enable it:
 kubectl patch networks.operator.openshift.io cluster --type merge -p '{"spec":{"deployKubeProxy": true}}'
 ```
 
+#### MKE
+
+If you are running MKE, you can disable `kube-proxy` as follows:
+
+Download the cluster configuration, set `kube_proxy_mode` to `disabled` and `kube_default_drop_masq_bits` to `true` and upload the cluster configuration.
```

Comment:
I think something along these lines would work a bit more smoothly:

Follow the step procedure in [Modify an existing MKE configuration](link) to download, edit, and upload your MKE configuration. During the editing step, add the following configuration:
```
config config
```